mp/src/game/client/sdk/hud/sdk_scoreboard.cpp

```diff
@@ -466,6 +466,18 @@ void CSDKScoreboard::UpdatePlayerInfo()
 		sdkPR->GetHighestGrenadeKillPlayer(),
 		sdkPR->GetHighestGrenadeKills()
 	);
+
+	Label *pTimeLimitLabel = dynamic_cast<Label *>(FindChildByName("TimeLimit"));
+	if (pTimeLimitLabel)
+	{
+		int iTimeElapsed = SDKGameRules()->GetMapElapsedTime() / 60;
+		int iTimeLimit = mp_timelimit.GetInt();
+
+		wchar_t wszNewLabel[100];
+		V_swprintf_safe(wszNewLabel, iTimeLimit > 0 ? L"Time: %i/%i minute(s)" : L"Time: %i minute(s)", iTimeElapsed, iTimeLimit);
```

Comment:
Load this string from i18n string.

Recommend rephrasing for clarity: "Play time: n minutes" vs "Time remaining: n minutes"